requirements.txt

```diff
@@ -13,3 +13,4 @@ jsonschema==2.5.1
 ansible==2.1.0
 coverage
 nose
+python-openstackclient==0.8.6
```

Comment:
Are nose and python-openstackclient really requirements? Or are they requirements for testing? If I'm running duffy or gce, do I really need either of those?